static/styles/portico/stats.css

```diff
@@ -1,5 +1,5 @@
 body {
-    font-family: "Source Sans 3", "Helvetica Neue", sans-serif !important;
+    font-family: "Source Sans 3", sans-serif !important;
     background-color: hsl(0, 0%, 98%);
 }
```

Comment:
I think this CSS file only appears when `portico.css` is also loaded, so we might be able to delete the `font-family` lines in this file.  But this change is an improvement in any case.